fashion-messaging-app-api/server.js

```diff
@@ -14,7 +14,8 @@ app.use(cors({
   credentials: true
 }));
 app.use(express.json()); 
-app.use(morgan())
+app.use(morgan('combined')); 
```

Comment:
I changed this because I got an error that Morgan deprecated because I didn't specify a format so I added 'combined' to fix the bug